images/server/install-packages.sh

```diff
@@ -95,6 +97,7 @@ if [[ "$1" =~ ^--.+$ ]]; then
             --samba-version-suffix=*) samba_version_suffix="${arg/*=/}" ;;
             --install-custom-repos=*) install_custom_repos="${arg/*=/}" ;;
             --package-selection=*) package_selection="${arg/*=/}" ;;
+            --install-custom-ceph) install_custom_ceph=1 ;;
```

Comment:
I am struggling to follow this the naming scheme / flow.

If we pass exactly one custom repo _or_ `install_custom_ceph` is not `1` (say 0) it will enable distro-ceph/chacra. Passing `--install-custom-ceph` sets `install_custom_ceph=1`.  

So it seems to me that `--install-custom-ceph` means "I (the user) am providing a ceph repo", right? It seems a bit confusing to me because I might take it to mean `the script is (or should be) providing a ceph repo`. 

How does `--ceph-from-custom` strike you? I tried to make it clear that ceph is coming from the list of custom repos *you, the user* are supplying. I'm open to other suggestions, but I think that `--install-custom-ceph` will just force me to read the code every time I need it to remind me how it works.